graphstorm-processing/pyproject.toml

```diff
@@ -41,6 +41,7 @@ pre-commit = "^3.3.3"
 types-mock = "^5.1.0.1"
 pylint = "~2.17.5"
 diff-cover = "^9.0.0"
+pytest-cov = "^6.0.0"
```

Comment:
Adding this to get coverage reports easier using `pytest --cov=graphstorm_processing tests/`

Specifically, I wanted to be able to see the dynamic contexts, which for each line of code shows us which test covers that line, see https://coverage.readthedocs.io/en/latest/contexts.html#dynamic-contexts

https://pytest-cov.readthedocs.io/en/stable/readme.html#usage

Reply:
Do we need to add dependencies requirements also in the doc?